peer/brontide.go

```diff
@@ -1993,9 +1993,34 @@ func newDiscMsgStream(p *Brontide) *msgStream {
 		// so that a parent context can be passed in here.
 		ctx := context.TODO()
 
-		// TODO(yy): `ProcessRemoteAnnouncement` returns an error chan
-		// and we need to process it.
-		p.cfg.AuthGossiper.ProcessRemoteAnnouncement(ctx, msg, p)
+		p.log.Debugf("Processing remote msg %T from peer=%x", msg,
+			p.IdentityKey())
+
+		errChan := p.cfg.AuthGossiper.ProcessRemoteAnnouncement(
+			ctx, msg, p,
+		)
+
+		// Start a goroutine to process the error channel for logging
+		// purposes.
+		//
+		// TODO(ziggie): Maybe use the error to potentially punish the
+		// peer depending on the error ?
+		go func() {
+			select {
+			case <-p.cg.Done():
+				return
+
+			case err := <-errChan:
+				if err != nil {
+					p.log.Errorf("Error processing remote "+
+						"msg %T from peer=%x: %v", msg,
+						p.IdentityKey(), err)
+				}
+			}
+
+			p.log.Debugf("Processed remote msg %T from peer=%x",
+				msg, p.IdentityKey())
```

Comment:
same here re format, would be nice to at least check it out locally first,
```
2025-06-04 00:10:50.673 [DBG] PEER brontide.go:2021: Peer([Bob]): Processed remote msg *lnwire.ChannelAnnouncement1 from peer=&{{[3b4391f 14005b9 1a42bd3 36d09a5 358992c 5f3053 8329c5 2d893c0 a5a0d 3013ef]} {[40b042 2040379 2cf4570 9402be 1c13ccf 29240ce 2e649b 3a4d5f1 25c0f56 341a18]}}
```

Reply:
Good point.